source/CSharp.BlankApplication/Program.cs

```diff
@@ -1,22 +1,31 @@
 ﻿using System;
 using System.Threading;
+using System.Diagnostics;
 
 namespace $safeprojectname$
 {
     public class Program
     {
         public static void Main()
         {
-          Thread.Sleep(1000);	// Temporary dirty fix to enable correct debugging session
-	  // Insert your code below this line
+            while (!Debugger.IsAttached) { Thread.Sleep(100); }    // Wait for debugger (only needed for debugging session)
```

Comment:
@MikroBusNet I believe that this line should be commented rather then being the default...
I got this from the issues @piwi1263 was having yesterday struggling to have the board run the app when not in VS debug session. 
The thing is: it's highly likely that a developer (in the eager to get the app blinking) overlooks the need to comment that line when deploying the app and will get frustrated in the process.
Care to comment? 😉

Reply:
I have no preference. You can comment the debug line.

Reply:
This does not help in adding comments in code., as they aren't read.